.editorconfig

```diff
@@ -0,0 +1,20 @@
+[*]
+indent_style = tab
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+
+[*.yml]
+indent_style = space
+indent_size = 2
+
+[*.py]
+indent_style = space
```

Comment:
```suggestion
indent_style = tab
```

Reply:
no. it's a python file

Reply:
Yes, it's python file and it should use tab, not shitty spaces